ovs_performance.py

```diff
@@ -123,6 +123,9 @@
 DEFAULT_STREAM_LIST               = '10, 1000, 10000, 100000, 1000000'
 DEFAULT_BRIDGE_NAME               = 'ovs_pvp_br0'
 DEFAULT_WARM_UP_TIMEOUT           = 360
+DEFAULT_DEST_MAC_ADDRESS          = '00:01:02:03:04:05'
```

Comment:
Can you keep the current default mac addresses, which is "00:00:02:00:00:00"

Reply:
sure can - that was a mistake.